tests/terraform/runner/test_runner.py

```diff
@@ -438,6 +444,15 @@ def test_provider_uniqueness(self):
         provider = next(check for check in result.passed_checks if check.resource == "aws.one-line")
         self.assertIsNotNone(provider.file_line_range)
 
+    def test_entire_resources_folder(self):
+        current_dir = os.path.dirname(os.path.realpath(__file__))
+        valid_dir_path = current_dir + "/resources/"
+        runner = Runner(db_connector=self.db_connector())
+        if isinstance(runner.db_connector, IgraphConnector):
+            result = runner.run(root_folder=valid_dir_path, external_checks_dir=None, runner_filter=RunnerFilter())
```

Comment:
can you please limit it to fixed set of policies, because otherwise you would need to adjust the amount passed/failed checks with each new policy added